### DISCUSSION TITLE
mspider/install/offline-upgrade/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3360] [Announcements]
# mspider/install/offline-upgrade/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/install/offline-upgrade/

**Comment by samzong** [comment]
导航的目录名称与文档名称不一致，一个是升级服务网格，一个是离线升级服务网格

**Reply by samzong** [reply]
<img width="735" alt="image" src="https://github.com/DaoCloud/DaoCloud-docs/assets/13782141/9259b85f-0b8f-4c2f-b948-ad6d209964e3">

**Reply by windsonsea** [reply]
Updated in PR #5317
